lib/events/codes.go

```diff
@@ -96,6 +96,8 @@ const (
 	SessionUploadCode = "T2005I"
 	// SessionDataCode is the session data event code.
 	SessionDataCode = "T2006I"
+	// SessionRecordingAccessCode is the session recording view data event code.
+	SessionRecordingAccessCode = "T2007I"
```

Comment:
we should clump the [T20XXX](https://github.com/gravitational/teleport/blob/master/lib/events/codes.go#L107) together, i think we made a mistake cutting it off in the middle so it was hard to see the next sequence